sql/mob_spawn_points.sql

```diff
@@ -45116,7 +45116,7 @@ INSERT INTO `mob_spawn_points` VALUES (17289634,'Antican_Lanista','Antican Lanis
 INSERT INTO `mob_spawn_points` VALUES (17289635,'Cactuar','Cactuar',10,-665.247,-7.970,-681.9,127);
 INSERT INTO `mob_spawn_points` VALUES (17289636,'Tulwar_Scorpion','Tulwar Scorpion',11,-718.999,-8.294,-632.175,127);
 INSERT INTO `mob_spawn_points` VALUES (17289637,'Phorusrhacos','Phorusrhacos',19,-685.635,-9.206,-701.860,126);
-INSERT INTO `mob_spawn_points` VALUES (17289638,'Picolaton','Picolaton',38,-501.578,-0.133,-46.108,221);
+INSERT INTO `mob_spawn_points` VALUES (17289638,'Picolaton','Picolaton',38,-504.039,-0.085,-37.812,221);
```

Comment:
Out of curiosity, how where this spawn points obtained?

In general, not this concrete one